docs/sources/setup-grafana/configure-security/_index.md

```diff
@@ -39,6 +39,12 @@ There are many firewall tools available, refer to the documentation for your spe
 
 Require all network requests being made by Grafana to go through a proxy server.
 
+Self-hosted reverse proxy options include:
```

Comment:
@chri2547 what do you think about this suggestion? I just want to make it clear that these are some examples, but we don't in any way prefer them over other proxy options.

```suggestion
Self-hosted reverse proxy options include but are not limited to:
```